src/tests/ftest/nvme/enospace.py

```diff
@@ -302,12 +306,13 @@ def test_enospace_time_with_fg(self):
         #Repeat the test in loop.
         for _loop in range(10):
             print("-------enospc_time_fg Loop--------- {}".format(_loop))
+            print(self.pool.pool_percentage_used())
```

Comment:
Minor: It's preferred to use self.log.info() instead of print()

Reply:
I will update in the next PR.